duckduckgo/macos-browser
# Fix warnings in unit tests target

Task/Issue URL: https://app.asana.com/0/1201037661562251/1201935272396242/f
Tech Design URL:
CC:

**Description**:
I fixed importing bridging headers, and I also enabled the commented out test case for making URLs from address bar strings.

**Steps to test this PR**:
1. Verify that compiling unit tests does not produce warnings 
1. Verify that unit tests pass
1. Verify that typing `https://duckduckgo.com/?q=search string` in the search bar results in searching for `search string`.

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
